### DISCUSSION TITLE
Help defining the Config file for my own dataset inside Databricks

### DISCUSSION THREAD

**Discussion by strangan** [RUCAIBox/RecBole#1055] [Q&A]
Dear RecBole Developers. 

Can you help me with my configuration file? I'm trying to train a Sequential recommender on my own dataset, but I'm having trouble getting past the configuration step.

I'm using a Python Databricks notebook (so that i have access to GPU) and i've created a dataframe that looks like the one below --- WORKFLOWID, ACTIONID & SEQUENCE.

![image](https://user-images.githubusercontent.com/7209421/143094799-d8770701-13fa-4601-b00e-28fc8a87437d.png)

It's a pandas dataframe that i have written to an inter file in the dbfs filesystem that's tab delimited.

This is how i'm defining my config file:

![image](https://user-images.githubusercontent.com/7209421/143095433-b996c0d7-6f36-4fad-89aa-3bf3cfa183f3.png)

paramter_dict = {
  'model': 'Caser',
  'dataset': 'nwc',
  'data_path': "dbfs:/FileStore/recbole/nwc_action_sequences.inter",
  'load_col': {'inter': ['WORKFLOWID', 'ACTIONID', 'SEQUENCE']},
  'USER_ID_FIELD': 'WORKFLOWID',
  'ITEM_ID_FIELD': 'ACTIONID',
  'TIME_FIELD': 'SEQUENCE',
  'MAX_ITEM_LIST_LENGTH': 3,
  'neg_sample': 'None'
}

from logging import getLogger
from recbole.config import Config
from recbole.data import create_dataset
from recbole.data.utils import get_dataloader
from recbole.utils import init_logger, init_seed, get_model, get_trainer, set_color

`config = Config(config_dict=paramter_dict)`

But i get the following error when I run the above statement

`ValueError: neg_sampling [{'uniform': 1}] should be None when the loss_type is CE.`

Here's a screenshot of the entire cell

![image](https://user-images.githubusercontent.com/7209421/143095236-aef88823-f0a2-4568-8ded-6553bd6f1ed2.png)

Can you point me in the right direction?

**Comment by Sherry-XLL** [comment]
@strangan Hello, you should set `neg_sampling: ~` when the loss_type is `CE`.
